### DISCUSSION TITLE
JSON: Failed to load the submitted data due to invalid formatting

### DISCUSSION THREAD

**Discussion by 0x1AE7F** [pocketbase/pocketbase#1381] [Q&A] [answered]
Hey!
Im trying to update a JSON field by calling
`await pb.collection('users').update($currentUser.id, '["a"]')`
which works on username and lots of other fields but when I try to use that on a JSON field I get  the following error:
```
{
  "errorDetails": "json: cannot unmarshal array into Go value of type map[string]interface {}",
  "errorMessage": "Failed to load the submitted data due to invalid formatting."
}
```
I already tried using `JSON.parse()` and `JSON.stringify()` to convert the string to an actual JSON object but that did not work either.. What am I doing wrong?

**Comment by ganigeorgiev** [comment, marked as answer]
You need to specify the field name you want to update, aka:

```js
await pb.collection('users').update($currentUser.id, {
    myJsonField: ["a"]
})
```

**Comment by 0x1AE7F** [comment]
nvm im completely stupid..
The line should be
`await pb.collection('users').update($currentUser.id, {"myJsonField":  '["a"]'})`

Sorry for the useless question
